### ISSUE TITLE
Why the input of network is torch.cat([masked_img, (1-mask)]).

### ISSUE THREAD

**Issue by LoveU3tHousand2** [NiFangBaAGe/CoordFill#12]
The input of in-painting network always was torch.cat([masked_img, mask]) in early works, why it is reversed in coordfill ? 
I  had spend a lot time to find this 'bug'...
Is there any reason?
